pkgs/applications/networking/irc/weechat/scripts/multiline/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchurl, substituteAll, PodParser }:
+
+stdenv.mkDerivation {
+  pname = "multiline";
+  version = "0.6.3";
+
+  src = fetchurl {
+    url = "https://raw.githubusercontent.com/weechat/scripts/945315bed4bc2beaf1e47f9b946ffe8f638f77fe/perl/multiline.pl";
+    sha256 = "1smialb21ny7brhij4sbw46xvsmrdv6ig2da0ip63ga2afngwsy4";
+  };
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "weechat";
    repo = "scripts"
    rev = "945315bed4bc2beaf1e47f9b946ffe8f638f77fe";
    sha256 = lib.fakeSha256;
  };
```

Reply:
This repository contains lots of scripts, but this package packages only one. Thus, imho sourcing the whole repository seems a bit over the top. Furthermore, other scripts are doing the same, e.g., `weechat-colorize_nicks`.

Reply:
You are right. I wanted to make it easier to update it. We can probably just keep it.